miso-web/pom.xml

```diff
@@ -13,6 +13,7 @@
   <properties>
     <skipITs>true</skipITs>
     <skipSTs>true</skipSTs>
+    <jacoco.skip>true</jacoco.skip>
```

Comment:
Please add documentation to the developer's guide for how to enable, and what to look for in the results